packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -128,6 +128,7 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
         <Link
           className={ css('ms-Breadcrumb-itemLink', styles.itemLink) }
           href={ item.href }
+          aria-current={item.isCurrentItem ? 'page' : null}
```

Comment:
Didn't know that @jspurlin , good info. do you know when 1.1 will replace 1.0? The other option here would be to change these to be the equivalent of tabs inside of a tablist, not in a navigation section with links